WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/BlogDashboardCardFrameView.swift

```diff
@@ -247,9 +272,17 @@ class BlogDashboardCardFrameView: UIView {
         static let ellipsisButtonPadding = UIEdgeInsets(top: 0, left: 8, bottom: 0, right: 8)
         static let buttonContainerStackViewPadding: CGFloat = 8
         static let mainStackViewTrailingPadding: CGFloat = 32
+        static let newButtonInsets = UIEdgeInsets(top: 4, left: 8, bottom: 4, right: 6).flippedForRightToLeft
```

Comment:
It's not a noticeable change, but upon closely inspecting the button and the Figma design, I think the below spacing could be more accurate:
```suggestion
        static let newButtonInsets = UIEdgeInsets(top: 2, left: 8, bottom: 2, right: 8)
```

|Xcode Inspector|Figma|
|-|-|
|<img width="300" alt="Screen Shot 2023-04-25 at 2 43 30 PM" src="https://user-images.githubusercontent.com/25306722/234281021-a72b54e8-5078-4e9b-b00f-87efe7f33521.png">|<img width="300" alt="Screen Shot 2023-04-25 at 2 44 25 PM" src="https://user-images.githubusercontent.com/25306722/234281034-f2df46d7-7fee-45e4-ad51-47589b5e824e.png">|